src/Groups/GAPGroups.jl

```diff
@@ -456,6 +456,8 @@ struct GroupConjClass{T<:GAPGroup, S<:Union{GAPGroupElem,GAPGroup}}
    CC::GapObj
 end
 
+Base.eltype(::Type{GroupConjClass{T,S}}) where {T,S} = S
+Base.eltype(::Type{GroupConjClass}) = GAPGroupElem
```

Comment:
why is this second one needed?

Reply:
If I do not write it, the output of `eltype(GroupConjClass)` would be `Any`. But we can decide that this is fine.

Reply:
The goal is not to have eltype return anything in particular for all types; but rather it is to make things like the result of `collect(FOO)` have a sensible type for as many kinds of `FOO` as possible. I don't think we need this method for that -- but I might be wrong. As such, better tests that test the the actual end goal would be nice (i.e., `@test typeof(collect(FOO)) == Vector{SOMETYPE}` for suitable FOO and SOMETYPE

Reply:
And put an `@inferred` in front of it.

Reply:
Goos point, thanks!

@GDeFranceschi see https://docs.julialang.org/en/v1/stdlib/Test/#Test.@inferred